yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,130 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
```

Comment:
Would suggest to have the size to write(inout parameter) and also the size written (return parameter)

Reply:
I don't find the purpose of sending back the size written to the caller.
Instead while printing "Data updated successfully" message, I can incorporate the no of bytes written.

Reply:
Also size as an input parameter is not required for us. Caller knows the number of bytes he sends. and Our code checks for the actual array size and compares it with the given input size. and takes action accordingly.